test/docker-compose.yml

```diff
@@ -19,7 +19,3 @@ services:
     volumes:
       - "./../:/go/src/github.com/cilium/cilium/"
     privileged: true
-  precheck:
-    extends:
-      service: base_image
-    command: "bash -c 'cd /go/src/github.com/cilium/cilium/; make ci-precheck || exit 1'"
```

Comment:
Seems this is the only usage of the `make ci-precheck` target, so it could be removed as well while at it.

Reply:
We can also remove this `test/docker-compose.yml` file by the looks of it.